src/modules/order/index.ts

```diff
@@ -15,6 +15,9 @@ import findOrderForSession from './findOrderForSession';
 import loadCart from './loadCart';
 import removeOrderItem from './removeOrderItem';
 import addOrderItem from './addOrderItem';
+import changeOrderInfo from './changeOrderInfo';
```

Comment:
Are these not picked up from their files?
I do prefer this way but think it might be registering twice

Reply:
This was done because its the standard for every other module. If we change this we should change them all